googleapiclient/discovery.py

```diff
@@ -82,6 +82,9 @@
 VARNAME = re.compile('[a-zA-Z0-9_-]+')
 DISCOVERY_URI = ('https://www.googleapis.com/discovery/v1/apis/'
```

Comment:
It's a part of the public API, so we can't remove it until we bump to 2.0.0, regardless of the fact that we ourselves are no longer using it.

Always strive for a minimal public API in a software library. :-\